src/main/api/2/studio-api-2.yaml

```diff
@@ -6330,6 +6330,182 @@ paths:
+              properties: 
+                items:
+                  type: array
+                  description: path(s) of content item(s)
+                  items:
+                    type: string
+                optionalDependencies:
+                  type: array
+                  description: optional (soft) dependencies
+                  items:
+                    type: string
+                publishingTarget:
+                  type: string
+                  description: target to publish content to
+                schedule:
+                  type: string
+                  format: datetime
+                  description: schedule when to publish content
+                submissionComment:
+                  type: string
+                  description: comment about publish request
+                sendEmailNotifications:
+                  type: boolean
+                  description: if true, send email notifications
+              required:
+                - items
+                - environment
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  response:
+                    $ref: '#/components/schemas/ApiResponse'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+
+  /workflow/publish:
+    post:
+      tags:
+        - workflow
+      summary: Publish content
+      description: 'Required permission "publish"'
+      operationId: workflowPublish
+      requestBody:
+        description: parameters for publish content
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties: 
+                items:
+                  type: array
+                  description: path(s) of content item(s)
+                  items:
+                    type: string
+                optionalDependencies:
+                  type: array
+                  description: optional (soft) dependencies
+                  items:
+                    type: string
+                publishingTarget:
+                  type: string
+                  description: target to publish content to
+                schedule:
+                  type: string
+                  format: datetime
+                  description: schedule when to publish content
+                submissionComment:
+                  type: string
+                  description: submission comment by author
+                reviewerComment:
+                  type: string
+                  description: Reviewer's comment
```

Comment:
Two different comments on the request?

We only have a single field on the publish dialog. Furthermore, the backend doesn't provide the submission comment by the author for us to include on the request when approving.

It helps to keep the two apis the same so perhaps we could rename to just `comment` and use that in both?

Reply:
Per API v1 docs, old API has 2 fields 
https://docs.craftercms.org/en/4.0/developers/projects/studio/api/workflow/go-live.html